test/externalTests/ens.sh

```diff
@@ -28,22 +28,23 @@ verify_input "$@"
 BINARY_TYPE="$1"
 BINARY_PATH="$2"
 
-function compile_fn { npx truffle compile; }
-function test_fn { npm run test; }
+function compile_fn { yarn build; }
+function test_fn { yarn test; }
 
 function ens_test
 {
-    local repo="https://github.com/ensdomains/ens.git"
+    local repo="https://github.com/ensdomains/ens-contracts.git"
     local branch=master
-    local config_file="truffle.js"
+    local config_file="hardhat.config.js"
```

Comment:
Why hardhat instead of truffle?

Reply:
Because they switched to Hardhat.

It's not the same repo really. The [ens](https://github.com/ensdomains/ens) repo was only ENS and was using Truffle. The [ens-contracts](https://github.com/ensdomains/ens-contracts) has contracts from the original ENS repo and from some other repos and has config written from scratch for Hardhat.